docs/src/design/aux_table/bitwise.md

```diff
@@ -88,20 +91,22 @@ The above constraints enforce that when $k_1 = 1$ , $a' = 16 \cdot a + \sum_{i=0
 
 ### Output aggregation
 
-To ensure correct aggregation of operations over individual bits, first we need to ensure that in the first row of every 8-row cycle, the value in column $z$ is exactly equal to the aggregated values of a bitwise operation applied to columns $a_0$, $a_1$, $a_2$, $a_3$, $b_0$, $b_1$, $b_2$, $b_3$. For an AND operation, the constraint enforcing this would look as follows:
+To ensure correct aggregation of operations over individual bits, first we need to ensure that in the first row, the aggregated output value of the previous row should be 0.
+$$
+k_0 \cdot z_p = 0
+$$
 
+Next, we need to ensure that for each row except the last, the aggregated output value must equal the previous aggregated output value in the next row.
 $$
-k_0 \cdot \left(z - \sum_{i=0}^3(2^i \cdot a_i \cdot b_i)\right) = 0
+k_1 \cdot \left(z - z'_p\right) = 0
 $$
 
-Lastly, we need to ensure that for all other rows, the value in the $z$ column is computed by multiplying the value from the previous row of the column by 16 and then adding it to the bitwise operation applied to the next set of bits of $a$ and $b$. This can be enforced with the following constraint:
+Lastly, we need to ensure that for all rows, the value in the $z$ column is computed by multiplying the previous value (from the current row) by 16 and then adding it to the bitwise operation applied to the row's set of bits of $a$ and $b$. This can be enforced with the following constraint:
```

Comment:
nit: It might be a bit clearer to say "... by multiplying the previous output value (from the $z_p$ column in the current row)..."